### ISSUE TITLE
Type '"AuthController.login"' is not assignable to type '"drive.local.serve"'.

### ISSUE THREAD

**Issue by finestgecko** [microeinhundert/radonis#1]
First of all, thank you for your efforts with this amazing project :tada: I think it's a considerable addition to the AdonisJS ecosystem and I'm really enjoying working with it.

I'm still learning how things work with Radonis. I copied the "old-school way" example from the [Forms](https://radonis.vercel.app/docs/forms) section in the docs, but I get a TypeScript error in Visual Studio Code on the `action` prop:

![image](https://user-images.githubusercontent.com/18103212/191131482-4b8467c3-3e4d-4194-a50e-2a5c461aad33.png)

```tsx
import { useFlashMessages, Form } from '@microeinhundert/radonis'

export default function Login({}: ILoginProps) {
  const { tx } = useTwind()

    return (
        <Form
              method="post"
              action="AuthController.login" // 👈 this prop
              reloadDocument
        >
            ...
        </Form>
    )
}
```

The route is registered:
```ts
Route.post('/auth/login', 'AuthController.login')
```

---

P.S. A quick suggestion&mdash;I think it would be useful if the docs could have a somewhat more gentle introduction to the Hydration concept. I know it's not a Radonis-specific concept but having never used hydration before, I'm not really sure where to apply it.

**Comment by microeinhundert** [comment]
Thank you. The type generation is currently not working as intended and will be fixed in the latest version coming out soon. Until then, a restart of the server should fix the problem and the Route should be available. If not, excluding the /tmp directory from TypeScript should disable the generated types. I will get back to you once it's fixed.

**Comment by microeinhundert** [comment]
I published a new version 1.9.2 with fixed type generation. There are a few things to keep in mind (your issue was one of them) which are now documented in the [docs](https://radonis.vercel.app/docs/type-generation). The dev server must be restarted manually after adding new routes to the application. This is a technical limitation which I hope to remove long term.

The Discussions here on GitHub are also now enabled for the repository, feel free to ask questions there.

**Comment by microeinhundert** [comment]
It seems like pnpm does not detect the changes I made to folder casing when updating dependencies. Deleting node_modules and reinstalling should also fix it, although I would find it odd if pnpm had such bug. It worked for me when testing with npm.

**Comment by microeinhundert** [comment]
I know what the problem is. I'm on a mac and macOS behaves differently with folder casing. Patch is coming very soon.

**Comment by microeinhundert** [comment]
It should be fixed now in version 1.9.3.
